### ISSUE TITLE
Lens error after fresh install on Windows

### ISSUE THREAD

**Issue by StefanPopa02** [lensapp/lens#8025]
**Describe the bug**
I just installed Lens Desktop on my machine and it fails when opening with "ERROR!!
No matching route for the current path: /extension/lensapp--lenscloud-lens-extension/welcomepage which is the welcomeUrl. This is a bug."

**To Reproduce**
1. Fresh install Lens Desktop on Windows 11
2. After opening the application I see the error

**Expected behavior**
The application should work without any errors as it's just installed.

**Screenshots**
![image](https://github.com/lensapp/lens/assets/60078395/6abf3da3-fd72-4e5f-82da-9fa41032f50c)
![image](https://github.com/lensapp/lens/assets/60078395/4ec6ca42-d061-4c1c-aa27-989374ee5d9c)
![image](https://github.com/lensapp/lens/assets/60078395/0b06c56b-1800-483c-a6c9-d4e0c0220e47)

**Environment (please complete the following information):**
- Lens Version: 2024.1.300751-latest
- OS: Windows 11 Pro 23H2
- Installation method (e.g. snap or AppImage in Linux): Installed using Lens Setup 2024.1.300751-latest.exe

**Kubeconfig:**
I didn't use any kubeconfig yet

**Comment by panuhorsmalahti** [comment]
It can't create a folder for this extension. Is there enough disk space? Or maybe are there some file permissions problems?

**Comment by StefanPopa02** [comment]
![image](https://github.com/lensapp/lens/assets/60078395/891bef1e-6fb6-476d-9b1d-68ac5c3909cc)
I have full permissions over this application. There is also no problem with disk space

**Comment by yuval955** [comment]
I had the same error on Ubuntu with the same Lens Desktop version as yours.
I removed Lens Desktop from my Ubuntu and deleted the **Lens** directory that was located in the **config** directory. It solved the issue.

**I did the following commands:**
apt remove lens
apt purge lens
rm -rf ~/.config/Lens

And then I installed the Lens and logged in again.
Translate the commands for Windows and I believe you will solve it.
